support/models/src/local/ModeSelectOverrides.ts

```diff
@@ -21,13 +21,12 @@ LocalMatter.children.push({
             ],
         },
 
-        // Spec defines as an enum but does not define values.  We could special case to generate an enum from the
-        // standard namespaces but for now we just set as an integer
+        // Spec defines as an enum8. We have the datatype "namespace" for exactly this enum, so let's use it.
         {
             tag: "attribute",
             name: "StandardNamespace",
             id: 0x1,
-            type: "uint16",
+            type: "namespace",
```

Comment:
We also need to update Semtag#namespaceId and SemanticNamespace#id so types match up

Reply:
The codegen tweak I mentioned this will require is to SemanticNamespaceFile.ts...  Needs to generate enum rather than raw number